experimental/devices/pca9685/pca9685.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package pca9685
+
+import (
+	"math"
+	"time"
+
+	"periph.io/x/periph/conn/physic"
+
+	"periph.io/x/periph/conn/i2c"
+)
+
+// PCA9685Address i2c default address
```

Comment:
There's a bit of inconsistency in the drivers for default I2C addresses, I think I'd prefer `I2CAddr` here, since PCA9685Address stutters.

https://blog.golang.org/package-names talks a bit about this.